### PR TITLE
address montonic and prediction size issues

### PR DESCRIPTION
**Pull request by MartinSJRogers**
This PR addresses the monotonic and prediction size issues. It tries to address these instead of using the previous PRs on this issue: #7 and #6

### CODE REVIEW

File: deepsensor/model/model.py [hunk 1]
```diff
@@ -918,29 +918,38 @@ def stitch_clipped_predictions(patch_preds, patch_overlap, patches_per_row, x1_a
                         """
                         if patch_x2_index[0] == data_x2_index[0]:
                             b_x2_min = 0
+                            # This line, as well as 933, 940 and 950 address the different shapes between the input and prediction
```
Comment (davidwyld) [inline deepsensor/model/model.py]:
```suggestion
                            # The +1 operations here and elsewhere in this block address the different shapes between the input and prediction
```

File: deepsensor/model/model.py [hunk 2]
```diff
@@ -918,29 +918,38 @@ def stitch_clipped_predictions(patch_preds, patch_overlap, patches_per_row, x1_a
...
+                            # TODO: Try to resolve this issue in data/loader.py by ensuring patches are perfectly square.
```
Comment (davidwyld) [inline deepsensor/model/model.py:922]: I'll look into this but let's merge this for now.

**Review by davidwyld — CHANGES_REQUESTED** [review]
Lets merge this, only one change suggested to a comment. Let's close the other PRs as well. Nice one.

**Review by MartinSJRogers — COMMENTED** [review]
Change to code description agreed.

**Review by davidwyld — APPROVED** [review]
:rocket:

**Comment by MartinSJRogers** [comment]
Happy to merging and close this PR as discussed. Thinking out loud: in dataloader.py perhaps there is a location where we can assert that patched tasks have an equal size in `x1` and `x2`?

**Pull request merged**